pages/about/AboutHandler.inc.php

```diff
@@ -309,7 +309,7 @@ function editorialTeamBio($args) {
 	/**
 	 * Display editorialPolicies page.
```

Comment:
Should have self-docs:

    @param $args array
    @param $request PKPRequest